source/updateCheck.py

```diff
@@ -520,7 +520,7 @@ def onInstallButton(self, evt):
 	def onPostponeButton(self, evt):
 		finalDest=os.path.join(storeUpdatesDir, os.path.basename(self.destPath))
 		try:
-			# #9825: behavior of os.rename(s) has changed in Python 3.
+			# #9825: behavior of os.rename(s) has changed in Python 3 (needed for file renames, possibly across drives in case of portable cop0y).

```

Comment:
The current comment in this PR says the behaviour has changed, but not how the behaviour changed. #9825 is not very illuminating, it would be good to save someone reading this code from having to go through a whole investigation, as I have now had to do. 

The top of issue #9825 says we get windows error 17. My best guess is that "error 17" is:

```
ERROR_NOT_SAME_DEVICE
    17 (0x11)
    The system cannot move the file to a different disk drive.
```

But it's not clear to me what situation causes this to happen. Reading through the stack trace on the issue I've noticed that it does confirm this is the error in the exception. I also notice that the two paths are directories, and one is a relative path.

I've also just noticed the link to the python bug (via a comment from Leonard on this PR)

Perhaps we can replace this comment with: 
```
# Python 3.7's `os.renames` function no longer allows copies across disk drives, it raises `OSError` with ERROR_NOT_SAME_DEVICE. 
# The extended description of this error is "The system cannot move the file to a different disk drive".
# This can cause an error when updating a portable copy of NVDA"
# See https://bugs.python.org/issue28356 for information about the change to os.rename / os.renames
```